programs/compressed-pda/src/lib.rs

```diff
@@ -17,6 +24,19 @@ pub mod psp_compressed_pda {
         Ok(())
     }
 
+    /// This function can be used to transfer sol and execute any other compressed transaction.
+    /// Instruction data is optimized for space.
+    pub fn execute_compressed_transaction2(
```

Comment:
Maybe instead of using "2", we can distinguish these two methods by renaming the previous one to `execure_compressed_transaction_cpi`?

Reply:
hm, what do you think about optimized?
The reason why I created both was to keep one unopinionated method so that we can apply another algorithm to pack instruction data in the future via cpi.

Reply:
I used 2 because I have seen this in solana programs for example create_mint2